apiVersion: events.k8s.io/v1
deprecatedCount: 4
deprecatedFirstTimestamp: "2023-03-08T11:50:02Z"
deprecatedLastTimestamp: "2023-03-08T11:50:17Z"
deprecatedSource:
  component: kubelet
  host: vrutkovs-mrmwt-master-1.c.openshift-gce-devel.internal
eventTime: null
kind: Event
metadata:
  creationTimestamp: "2023-03-08T11:50:02Z"
  managedFields:
  - apiVersion: v1
    fieldsType: FieldsV1
    fieldsV1:
      f:count: {}
      f:firstTimestamp: {}
      f:involvedObject: {}
      f:lastTimestamp: {}
      f:message: {}
      f:reason: {}
      f:source:
        f:component: {}
        f:host: {}
      f:type: {}
    manager: kubelet
    operation: Update
    time: "2023-03-08T11:50:17Z"
  name: apiserver-6c795f96d5-wjvbx.174a6f507f48800c
  namespace: openshift-oauth-apiserver
  resourceVersion: "16484"
  uid: 871e7344-d89b-4f28-9032-ed7093f7a910
note: "Readiness probe error: Get \"https://10.128.0.18:8443/readyz\": dial tcp 10.128.0.18:8443:
  connect: connection refused\nbody: \n"
reason: ProbeError
regarding:
  apiVersion: v1
  fieldPath: spec.containers{oauth-apiserver}
  kind: Pod
  name: apiserver-6c795f96d5-wjvbx
  namespace: openshift-oauth-apiserver
  resourceVersion: "11218"
  uid: e92cec09-11d2-4ea7-bc1b-e3483ccb7451
type: Warning
